### PR TITLE
HOTT-1391 Add browse simulation

### PR DESCRIPTION
**Pull request by alexdesi**
Add browse simulation to simulate the browsing action: 
Load Section > Load chapter > Load heading > Load Commodity

### CODE REVIEW

File: src/test/resources/browse-commodities.csv [hunk 1]
```diff
@@ -0,0 +1,10001 @@
+section,chapter,heading,commodity
```
Comment (alexdesi) [inline src/test/resources/browse-commodities.csv:1]: These are 10,000 entries grabbed randomly from the commodities.

**Comment by alexdesi** [comment]
Screenshot 2022-03-23 at 17 40 26

File: src/test/resources/browse-commodities.csv (hunk 1 above)
Comment (willfish) [reply src/test/resources/browse-commodities.csv:1]: I don't think chapters and sections are interesting. Maybe just add subheadings to this and quota search

Comment (willfish) [reply src/test/resources/browse-commodities.csv:1]: I'd probably pull out the counts of ancestors/children and do a spread of a 1000 with varying counts of related goods nomenclature. 10k is overkill/just noise re sampling

Comment (alexdesi) [reply src/test/resources/browse-commodities.csv:1]: Chapters and sections are used in the CSV to simplify the code, it just uses a bit more disk space.
I agree 10k, are maybe too many, we can reduce them to 1k.

I've added tests for API quota in a separate test, and tests for the search too.

**Review by willfish — APPROVED** [review]
:sheep: :it: :tada:

**Pull request merged**